templates/seL4SharedDataWithCaps.template.c

```diff
@@ -52,12 +52,12 @@
 /*? macros.shared_buffer_symbol(sym=dataport_symbol_name, shmem_size=shmem_symbol_size, page_size=page_size) ?*/
 
 /*- set arch_name = obj_space.spec.arch.capdl_name() -*/
-/*- if arch_name in ['ia32', 'x86_64'] -*/
+/*- if arch_name in ['aarch32', 'aarch64', 'arm_hyp', 'ia32', 'x86_64'] -*/
```

Comment:
Are you sure this works? I find `capdl_name()` implemented in `capdl/python-capdl-tool/capdl/util.py` and it returns `arm11` for `ARM32Arch`, independent of `hyp` being `true`or `false`. Thus, I wonder if this should check `options.architecture` instead of `obj_space.spec.arch.capdl_name()`?